src/tools/rustfmt/src/expr.rs

```diff
@@ -345,6 +345,7 @@ pub(crate) fn format_expr(
         // Style Guide RFC for InlineAsm variant pending
         // https://github.com/rust-dev-tools/fmt-rfcs/issues/152
         ast::ExprKind::InlineAsm(..) => Some(context.snippet(expr.span).to_owned()),
+        ast::ExprKind::OffsetOf(..) => Some(context.snippet(expr.span).to_owned()),
```

Comment:
Does that mean "no formatting"?

Reply:
Yes, but these arms aren't even called according to a comment in a later arm so I moved it to that arm.